cmd/mt-parrot/main.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import (
+	"github.com/grafana/metrictank/cmd/mt-fakemetrics/out"
+	"github.com/grafana/metrictank/cmd/mt-fakemetrics/out/gnet"
+	"github.com/grafana/metrictank/logger"
+	"github.com/grafana/metrictank/schema"
+	"github.com/raintank/met/statsd"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"time"
+)
+
+var (
+	gatewayAddress            string
+	gatewayKey                string
+	orgId                     int
+	partitionCount            int32
+	partitionMethodString     string
+	artificialMetricsInterval time.Duration
+	queryInterval             time.Duration
+	logLevel                  string
+
+	partitionMethod schema.PartitionByMethod
+	gateway         out.Out
+)
+
+func init() {
+	parrotCmd.Flags().StringVar(&gatewayAddress, "gateway-address", "http://localhost:6059", "the url of the metrics gateway")
+	parrotCmd.Flags().StringVar(&gatewayKey, "gateway-key", "", "the bearer token to include with gateway requests")
+	parrotCmd.Flags().IntVar(&orgId, "org-id", 1, "org id to publish parrot metrics to")
+	parrotCmd.Flags().Int32Var(&partitionCount, "partition-count", 8, "number of kafka partitions in use")
+	parrotCmd.Flags().StringVar(&partitionMethodString, "partition-method", "bySeries", "the partition method in use on the gateway, must be one of bySeries|bySeriesWithTags|bySeriesWithTagsFnv")
+	parrotCmd.Flags().DurationVar(&artificialMetricsInterval, "artificial-metrics-interval", 5*time.Second, "interval to send metrics")
```

Comment:
5s is an uncommon interval. 10s is a more useful default I think.

Reply:
ef8e98fac33b8fd13ea70cf83d1af74d1e3a3789